src/main/java/jenkins/branch/OrphanedItemsExtraCleanupProperty.java

```diff
@@ -34,14 +34,8 @@
 import hudson.ExtensionList;
 import hudson.Util;
 import hudson.XmlFile;
-import hudson.model.Action;
-import hudson.model.Cause;
-import hudson.model.CauseAction;
-import hudson.model.Items;
-import hudson.model.PeriodicWork;
+import hudson.model.*;
```

Comment:
You need to change your IDE settings so that it doesn't add/optimizes to `*` imports